docs/modules/ROOT/pages/metrics.adoc

```diff
@@ -0,0 +1,1681 @@
+= Metrics
```

Comment:
Can we change the title to "List of Hazelcast Metrics"?

Reply:
Made the requested change